docs/data/material/getting-started/overview/overview.md

```diff
@@ -0,0 +1,26 @@
+# Material UI - Overview
+
+<p class="description">Material UI is library of React UI components that implements Google's Material Design system.</p>
+
+## Introduction
+
+Material UI is an open-source React component library that implements [Material Design](https://material.io/).
+
+It includes a comprehensive collection of prebuilt components that are ready for production right out of the box.
+
+Material UI is beautiful by default, and it features a suite of customization options that make it easy to implement your own custom design system on top of our components.
+
```

Comment:
This is looking excellent, nice work as always, Sam! Guess that my only suggestion would be: should we add a section/paragraph mentioning _which_ version of Material Design we currently implement (which is v2) and what are our plans/perspectives to deal with v3?

Reply:
Good idea! We can add an info callout. Speaking of which—what is the plan for v3? 😅

Reply:
To mostly ignore it :D

Reply:
Roger that!